.github/workflows/test-python.yml

```diff
@@ -67,8 +67,8 @@ jobs:
 
       - name: maturin venv Build
         working-directory: python
-        run: uv run --no-project maturin develop
```

Comment:
The `--no-project` was here because I thought whenever we ran `uv sync` or `uv run`, uv would automatically build the current project, which would compile the rust code and take a while.

Reply:
Hmm, I though if the [working directory](https://github.com/astral-sh/setup-uv?tab=readme-ov-file#working-directory) is set to `python/` already, then it should only compile what's inside the `python/` folder? Don't we need to compile the rust code anyway before compiling the pyo3 code? A little confused here.

Reply:
It's not the cargo project that's the issue. It's the uv project. We don't want uv automatically building the rust parts of async-tiff in release mode.